web/src/main/java/com/futurewei/alcor/web/entity/dataplane/GoalStateJsonNB.java

```diff
@@ -0,0 +1,94 @@
+package com.futurewei.alcor.web.entity.dataplane;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.common.constants.Common;
```

Comment:
A side topic: I think we should remove alcor/common/constants/Common.java, which is an auto-generated protobuf file from schema/proto3/common.java.